app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -14,6 +14,13 @@ namespace crow {
 // Otherwise single (slow) clicks would easily trigger scrolling.
 const XrDuration kEyeTrackingScrollThreshold = 250000000;
 
+// Threshold to consider a trigger value as a click
+// Used when devices don't map the click value for triggers;
+const float kControllerClickThreshold = 0.25f;
+// Threshold to consider a pinch as a click. Hand tracking is not as precise as controllers, that's
+// why we need to be extra careful in order not to generate false positives.
+const float kPinchThreshold = 0.91f;
```

Comment:
The implementation is good but maybe this value is too low and we might run into the opposite problem. Perhaps a value like `0.5` might be more reliable.

Reply:
Have you tested it? I tried and it's very reliable, I didn't get any false positive

Reply:
As a matter of fact, it's difficult to see the improvement so I was going to suggest to reduce it. At least the Meta browse requires less pressure on the trigger.